packages/rx_bloc_cli/lib/src/commands/create_command.dart

```diff
@@ -371,6 +392,10 @@ class CreateCommand extends Command<int> {
       'Feature Widget Toolkit Showcase',
       arguments.enableWidgetToolkitFeature,
     );
+    _usingLog(
+      'Feature Facebook Authentication',
+      arguments.enableFacebookAuth,
+    );
```

Comment:
The word "feature" introduces confusion. We add social authentications as libraries to feature_login.